app/src/components/PlaybackControl.tsx

```diff
@@ -0,0 +1,266 @@
+    };
+
+    /**
+     * @desc shows a message using snackbar
+     * @param text {string}
+     */
+    showMessage = (text: string): void => {
+        const { enqueueSnackbar } = this.props;
+        enqueueSnackbar(text, { variant: 'warning', anchorOrigin: { vertical: 'bottom', horizontal: 'left' } });
+    };
+
+    /**
+     * @desc Stops the websocket from processing more information.
+     */
+    clearPlayback = (): void => {
+        if (this.playbackWS) this.playbackWS.close();
+    };
+
+    render() {
+        const {
+            classes, playback, selectedDateTimeNano, selectedInstrument,
+        } = this.props;
+        const { selectedUnit, unitSpeed } = this.state;
+        let disabledButtons: boolean = true;
+        if (selectedDateTimeNano.neq(0) && selectedInstrument.length !== 0) {
+            disabledButtons = false;
+        }
+        return (
+            <div
+                id={'playbackSection'}
+                className={classes.centerContent}
+            >
+                <div
+                    id={'playButton'}
+                    onClickCapture={() => { if (!playback && !disabledButtons) this.handlePlayOrderBook(); }}
+                    className={playback ? classNames(classes.button, classes.marginRight, classes.selectedButton)
+                        : classNames(classes.button, classes.marginRight)}
+                >
+                    PLAY
+                </div>
+                <div
+                    id={'pauseButton'}
+                    onClickCapture={() => { if (!disabledButtons) this.handlePauseOrderBook(); }}
+                    className={classNames(classes.button, classes.marginRight)}
+                >
+                    PAUSE
+                </div>
+                <FormControl>
+                    <InputLabel id={'UnitSelectorLabel'}>Unit</InputLabel>
+                    <Select
+                        id={'unitSelector'}
+                        labelId={'UnitSelectorLabel'}
+                        value={selectedUnit}
+                        onChange={this.handleUnitChange}
+                        className={classNames(classes.selectUnitInput, classes.marginRight)}
+                        disabled={playback}
+                    >
+                        {TIME_UNITS.map(time => {
+                            return (
+                                <MenuItem
+                                    key={`menuitem-${time}`}
+                                    value={time}
+                                >
+                                    {time}
+                                </MenuItem>
+                            );
+                        })}
+                    </Select>
+                </FormControl>
+                <TextField
+                    id={'unitSpeedSelector'}
+                    type={'number'}
+                    value={unitSpeed}
+                    onChange={this.handleUnitSpeedChange}
+                    label={'# of Units'}
```

Comment:
Can you make this a dynamic label based on the unit selected? I don't feel this is descriptive enough